packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -103,6 +117,20 @@ class Config(_BaseConfig):
                 },
                 # environments
                 {"name": "env", "type": "string", "value": ["DISPLAY=:0"]},
+                # SEE 'simcore.service.settings' label annotations for simcore/services/dynamic/jupyter-octave-python-math:1.6.5
+                {"name": "ports", "type": "int", "value": 8888},
+                {
+                    "name": "constraints",
+                    "type": "string",
+                    "value": ["node.platform.os == linux"],
+                },
+                {
+                    "name": "resources",
```

Comment:
that is how the python docker sdk works yes..

Reply:
Right, taking from https://docs.docker.com/engine/api/v1.41/#operation/ServiceCreate

![image](https://user-images.githubusercontent.com/32402063/168030613-9217dfc8-a066-4561-88ca-5b8309338939.png)